swim/labels.go

```diff
@@ -1,5 +1,48 @@
 package swim
 
+import (
+	"errors"
+	"strings"
+)
+
+var (
+	// labelsBannedKeyCharacters keeps a list of all banned occurances for label keys
+	labelsBannedKeyCharacters = []string{
+		// the equal sign is used in label checksum calculation. To prevent
+		// unintended or forced collisions on the checksums of labels we do not
+		// allow the equal sign to be used in the key portion of a label
+		"=",
```

Comment:
I don't agree. `=` is only blacklisted for keys and it doesn't seem useful for a key.